conf/main.yaml

```diff
@@ -1,30 +1,90 @@
-analyzers: [trajectories, miscellaneous]
+analyzers:
+- trajectories
+- miscellaneous
 auto-compactify: 0
 clear-popeye-output-on-entry-change: true
-collections-dir: ./collections
-default-lang: en
-default-notation: en
+collections-dir: /home/thomas/Schach
```

Comment:
It seems that aside from the format, there are some meaningful changes that override the default values in this pull request.